simln-lib/README.md

```diff
@@ -0,0 +1,13 @@
+# sim-ln
+
+A Lightning Network simulation library that enables testing and analysis of payment routing behavior in a controlled environment.
+
+## Overview
+
+sim-ln provides a framework for simulating Lightning Network payment routing behavior, allowing developers and researchers to:
+
+- Create simulated Lightning Network topologies
```

Comment:
This sounds to me like sim-ln allows you to spin up different channel graphs, which isn't the case. Could we change this bullet point to something about specific or random traffic patterns?